src/table/components/virtualized-table/Cell.tsx

```diff
@@ -19,32 +21,38 @@ const Cell = ({ columnIndex, rowIndex, style, data }: CellProps) => {
   const { rowsInPage, columns, bodyStyle } = data;
   const datum = rowsInPage[rowIndex]?.[`col-${columnIndex}`];
 
-  if (datum) {
+  const { handleMouseDown, handleMouseOver, handleMouseUp, selectionStyling } = useSelector(datum, {
+    backgroundColor: bodyStyle.backgroundColor,
+  } as CellStyle);
+
+  if (typeof datum === 'object') {
     return (
       <div
         style={{
           ...style,
+          ...bodyStyle,
+          ...selectionStyling,
           display: 'flex',
           alignItems: 'center',
-          borderColor: bodyStyle.borderColor,
-          borderStyle: 'solid',
```

Comment:
I have remove borderStyle from the styling-utils. There are no borders now when I am testing it

Reply:
Fixed that now